types/node/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Node.js 10.12
+// Type definitions for Node.js 11.6
```

Comment:
```suggestion
// Type definitions for Node.js 11.8
```
`timeout` option in `tls.connect()` was added in 11.8